client/src/www/model/platform_error.ts

```diff
@@ -14,6 +14,12 @@
 
 import {CustomError} from '@outline/infrastructure/custom_error';
 
+import * as errors from './errors';
+
+// TODO(fortuna): Remove PlatformError from the model.
+// PlatformError is an implementation detail. It does not belong in the model.
+// It's also about serialization, we should not use it in application code.
```

Comment:
👍 As we have a unified `MethodChannel` now, we can convert the errors there, without exporting any internal implementations.